modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -42,6 +77,26 @@ Changing the store format requires that:
 
 Changing store format can be a time-consuming operation, depending on the size and complexity of the data in the database. Performance heavily depends on the speed of the disk and the amount of available memory.
 
+=== Verify the store format
+
+You can verify the store format of a database using the following Cypher:
+
+[source,cypher]
+----
+SHOW DATABASES YIELD name, store
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------+
+| name      | store                 |
++-----------------------------------+
+| "blockdb" | "block-block-2b.1"    |
```

Comment:
This is the beta format which isn't supported since 5.14 and I don't think we mention here. The correct format is `block-block-1.1`